requirements-dev.txt

```diff
@@ -19,10 +19,10 @@ requirements-parser
 # requirements-dev.txt
 # linting.yml
 # .pre-commit-config.yaml
-black==23.9.1
+black==23.10.0
```

Comment:
Please read the comment above this change

Reply:
This one isn't particularly different anyway, so I'll just take it out of this PR.